cli/tests/integrations/test_marathon_debug.py

```diff
@@ -0,0 +1,152 @@
+import contextlib
+import os
+import pytest
+
+from .common import (app, exec_command, pod)
+from .test_marathon import (_list_tasks)
+
+_PODS_ENABLED = 'DCOS_PODS_ENABLED' in os.environ
+
+
+@pytest.mark.skipif(not _PODS_ENABLED, reason="Requires pods")
+def test_debug_list():
+    with _stuck_app():
+        returncode, stdout, stderr = exec_command(
+            ['dcos', 'marathon', 'debug', 'list'])
+
+        assert returncode == 0
+        assert stderr == b''
+
+        decoded = stdout.decode()
+        assert 'OVERDUE' in decoded
+        assert '/stuck-sleep' in decoded
+        """There should be a line of the following format in the stdout:
+        /stuck-sleep $since 9 True 4 3 $last_unsued_offer $last_used_offer
+        To avoid formatting issues, whitspaces are ignored.
+        Explanation:
+        9 Instances should be launched and $since timestamp ends with `Z`.
+        """
+        assert 'Z9' in decoded.replace(' ', '')
+
+
+@pytest.mark.skipif(not _PODS_ENABLED, reason="Requires pods")
+def test_debug_list_pod():
```

Comment:
I added a regular expression for this line checking all non dynamical parts of this table. Is this enough, what do you think?